src/integrationtest/python/TextAttribute_tests.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from JanusGraphContainer import JanusGraphContainer
+from janusgraph_python.driver.ClientBuilder import JanusGraphClient
+from gremlin_python.structure.graph import Graph
+from janusgraph_python.core.attribute.Text import Text
+
+
+class TestTextAttributes(unittest.TestCase):
+    def setUp(self):
+        self.container = JanusGraphContainer()
+
+        self.client = JanusGraphClient().connect(host=self.container.get_host_ip(), port="8182",
+                                                 traversal_source="gods_traversal").get_connection()
+
+        self.container.start()
+        self.g = Graph().traversal().withRemote(self.client)
+
+    def tearDown(self) -> None:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [invalid syntax](https://app.codacy.com/manual/JanusGraph/janusgraph-python/pullRequest?prid=4823236)

Reply:
Can be ignored.